src/OMSimulatorLib/System.cpp

```diff
@@ -389,6 +393,12 @@ oms_status_enu_t oms::System::exportToSSD(pugi::xml_node& node) const
   if (oms_status_ok != element.getGeometry()->exportToSSD(node))
     return logError("export of system ElementGeometry failed");
 
+  // export top level system connectors
+  pugi::xml_node connectors_node = node.append_child(oms::ssp::Draft20180219::ssd::connectors);
```

Comment:
This is because, A top level system can have connectors and parameter bindings, that is why it is moved there, otherwise only subsystem connectors are exported